Xamarin.RangeSlider.Droid/RangeSliderControl.cs

```diff
@@ -562,6 +567,22 @@ public void SetThumbShadowPath(Path thumbShadowPath)
             _thumbShadowPath = thumbShadowPath;
         }
 
+        public void SetCustomThumbImage(Bitmap customThumbImage) 
+        {
+            ThumbImage = customThumbImage;
+            UpdateThumbValues();
+        }
+
+        public void SetCustomThumbPressedImage(Bitmap customThumbImage) 
+        {
+            ThumbPressedImage = customThumbImage;

```

Comment:
`UpdateThumbValues` should be here and in the next method as well as there is no specific order in which these methods must be called.

Reply:
ok done